circumstellar-habitable-zone-simulator/src/timeline.jsx

```diff
@@ -19,15 +118,45 @@ export default class CSHZTimeline extends React.Component {
                 <h2>Timeline and Simulation Controls</h2>    
             </div>
             <div>
-                Time since star system formation:
+                Time since star system formation: {typeof this.props.starMassIdx == 'number' && (
```

Comment:
Minor style point -- this comparison could be triple-equals to reduce ambiguity. Same with the comparison on line 132 below.

Reply:
Got it, updated.